ui/src/config/section/compute.js

```diff
@@ -458,10 +458,10 @@ export default {
         return fields
       },
       filters: () => {
-        const filters = ['managed', 'unmanaged']
+        const filters = ['cloud.managed', 'external.managed']
         return filters
       },
-      details: ['name', 'description', 'zonename', 'kubernetesversionname', 'autoscalingenabled', 'minsize', 'maxsize', 'size', 'controlnodes', 'cpunumber', 'memory', 'keypair', 'associatednetworkname', 'account', 'domain', 'zonename', 'managed', 'created'],
+      details: ['name', 'description', 'zonenclusame', 'kubernetesversionname', 'autoscalingenabled', 'minsize', 'maxsize', 'size', 'controlnodes', 'cpunumber', 'memory', 'keypair', 'associatednetworkname', 'account', 'domain', 'zonename', 'clustertype', 'created'],
```

Comment:
sorry @vishesh92 just noticed this, is this a typo? `zonenclusame` ?

Reply:
My bad. Nice catch @Pearl1594